src/main/java/seedu/lifeasier/commands/AddNotesCommand.java

```diff
@@ -17,8 +17,18 @@ public AddNotesCommand(String title) {
     public void execute(Ui ui, NoteList notes, TaskList tasks) {
         ui.showNoteTitleMessage();
         String noteTitle = ui.readCommand();
+
+        while (noteTitle.trim().length() == 0) {
```

Comment:
Maybe in the while loop you can switch it to a variable called: isEmpty or something instead, then you update that variable through a method which returns true when noteTitle.trim().length() == 0